aider/resources/model-metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-    "deepseek-reasoner": {
+    "deepseek/deepseek-reasoner": {
```

Comment:
The confusing part here is that upstream has:

https://github.com/BerriAI/litellm/blob/main/model_prices_and_context_window.json#L5433

but https://aider.chat/docs/config/adv-model-settings.html does point out:

_Use a fully qualified model name with a provider/ at the front in the .aider.model.metadata.json file. For example, use deepseek/deepseek-chat, not just deepseek-chat. That prefix should match the litellm_provider field._